src/data/ff/ff.h

```diff
@@ -191,6 +192,13 @@ class Forcefield
     void assignAtomType(const ForcefieldAtomType &ffa, SpeciesAtom &i, CoreData &coreData, bool setSpeciesAtomCharges) const;
     // Assign intramolecular parameters to the supplied Species
     bool assignIntramolecular(Species *sp, int flags = Forcefield::GenerateImpropersFlag) const;
+
+    /*
+     * Pair Potential Overrides
+     */
+    public:
+    // Add or Return overrides for current forcefield
+    virtual std::vector<PairPotentialOverride> pairPotentialOverrides() { return {}; };
```

Comment:
Tweaking the comment a bit, and I'd rather see the function definition itself in the cpp file, even though it is a trivial one.
```suggestion
    // Return available pair potential overrides
    virtual std::vector<PairPotentialOverride> pairPotentialOverrides();
```